boefjes/boefjes/plugins/kat_adr_finding_types/main.py

```diff
@@ -0,0 +1,15 @@
+from typing import List, Tuple, Union
+
+import requests
+
+from boefjes.job_models import BoefjeMeta
+
+FINDING_TYPES_JSON_LOCATION = (
+    "https://raw.githubusercontent.com/minvws/nl-kat-coordination/finding-types-in-octopoes/adr_finding_types.json"
```

Comment:
This URL points to this branch, I don't think that's what we want (after this PR the branch will be deleted). Maybe point to `main`, or the current release branch?

Reply:
Yeah you're right, but for testing purposes its a little bit a chicken-and-egg verhaal haha

Reply:
This is a public interface now, so we should also think about backwards compatibility. I think there are two options:

- We can point to the release branch or tagged release to make sure we don't  break older versions of OpenKAT, but then what's the point in fetching this from github? We could just use the version included in OpenKAT.
- We can point to main, but then we have to make sure that any changes are backwards compatible. The change we just did with renaming KAT-TLS-1.1-AND-1.2-SUPPORT would break existing OpenKAT installations that would still use the older name. We can also never delete/rename fields. If we go this route I think we shoud include a version number so we have the option to do breaking changes or clean up cruft whilekeeping support for older OpenKAT versions for the same reason an API should also always include a version.

I think it would also be better to move the json files into a subdirectory instead of cluttering the root with them.

I think this data doesn't change that much in a breaking way that we need to point to a release branch or tag, so main would be fine if we version it so that we have the option to do breaking changes. So I suggest we we move it to `data/adr_finding_types/v1.json` and `data/kat_finding_types.json/v1.json`

Reply:
Ideally, in the future This list of Findings would be something that lives on a community maintained location (albeit with moderation), like openkat.nl. Findings that show up in your tree from Boefjes that we run can have various locations which provide them with up-to-date remedies and scores. Providing those scores only inside the application makes it harder to stay up to date with those changes.
I don't think we should bind the url to a specific version of OpenKAT, as packaged. But we should bind it to a specific version of this 'api'.
Eg, link to "https://raw.githubusercontent.com/minvws/nl-kat-coordination/finding-types-in-octopoes/adr_finding_types-v1json"
Which we can eventually supersede by:
"https://raw.githubusercontent.com/minvws/nl-kat-coordination/finding-types-in-octopoes/adr_finding_types-v2.json"
Or 
"https://openkat.nl/sources/adr_finding_types-v2.json"

Reply:
Since finding definitions indeed change alot per release, i agree with @praseodym suggestion. Doing it per api version would cause a lot of breaks.

Reply:
Created #1120 for this